azkaban-web-server/build.gradle

```diff
@@ -26,7 +26,7 @@ node {
     npmVersion = '5.6.0'
 
     // Base URL for fetching node distributions (change if you have a mirror).
-    distBaseUrl = 'https://nodejs.org/dist'
+    distBaseUrl = 'https://direct.nodejs.org/dist/'
```

Comment:
why is this change needed ?

Reply:
Get a Proxy error during building of azkaban-webserver.

Reply:
https://github.com/srs/gradle-node-plugin/issues/292
Faced this issue.